### ISSUE TITLE
[feature request] dark mode for the stylebot editor UI

### ISSUE THREAD

**Issue by wadawada** [ankit/stylebot#539]
dark mode for option page and editor UI

**Comment by probablytukars** [comment]
no update on this?

**Comment by dannycolin** [comment]
Hey @ankit,

Would you accept a PR for this feature. I've already implemented a dark mode in a few addons like Privacy Badger and Multi-Account Container so it shouldn't be to hard to port the logic I used in this addon.

**Comment by ankit** [comment]
@dannycolin yes, that will be awesome. I will happy to accept a PR for this.

**Comment by iamMG** [comment]
> Hey @ankit,
> 
> Would you accept a PR for this feature. I've already implemented a dark mode in a few addons like Privacy Badger and Multi-Account Container so it shouldn't be to hard to port the logic I used in this addon.

Pls do so! The UI is too frickin' white! :)

**Comment by SysAdminDoc** [comment]
Any update on darkmode for the UI? It is like getting flashbanged every time I open it.

**Comment by ChaliceChore** [comment]
@ankit since you have resumed active development, I hope you treat this issue as a priority one. :)

**Comment by ankit** [comment]
@ChaliceChore yes, it's coming soon. I am actively working on giving the UI a fresh coat of paint with dark mode support
